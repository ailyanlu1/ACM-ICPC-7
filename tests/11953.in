7
4
x...
..x.
@.@.
....
2
..
x.
4
x.@.
..x.
@.@.
..x.
4
xxxx
....
@.@.
@.x.
1
x
1
@
2
xx
..
