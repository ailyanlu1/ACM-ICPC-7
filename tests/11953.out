Case 1: 2
Case 2: 1
Case 3: 2
Case 4: 2
Case 5: 1
Case 6: 0
Case 7: 1
